pkg/apis/core/types.go

```diff
@@ -3575,30 +3575,33 @@ type ServiceSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/services-networking/service/
 	Selector map[string]string
 
-	// ClusterIPs identifies all the ClusterIPs assigned to or requested for  this
-	// service. ClusterIPs are assigned or reserved  based on the values of
-	// Service.Spec.IPFamilies. Maximum of two entries (dualstack IPs) are
+	// ClusterIPs identifies all the ClusterIPs assigned to this
+	// service. ClusterIPs are assigned or reserved based on the values of
+	// service.spec.ipFamilies. A maximum of two entries (dual-stack IPs) are
 	// allowed in ClusterIPs. The IPFamily of each ClusterIP must match
-	// values provided in Service.Spec.IPFamilies
+	// values provided in service.spec.ipFamilies
 	// +optional
 	ClusterIPs []string
 
-	// IPFamilies identifies all the IPFamilies assigned to or requested for
-	// this Service. If value was not provided for IPFamilies it will be defaulted
-	// based on the cluster configuration and the value of Service.Spec.IPFamilyPolicy.
-	// Maximum of two values (dualstack IPFamilies) are allowed in IPFamilies.
+	// IPFamilies identifies all the IPFamilies assigned for this Service. If a value
+	// was not provided for IPFamilies it will be defaulted based on the cluster
+	// configuration and the value of service.spec.ipFamilyPolicy. A maximum of two
+	// values (dual-stack IPFamilies) are allowed in IPFamilies. IPFamilies field is
+	// conditionally immutable it allows for adding another IPFamily or removing
```

Comment:
"immutable" is weird here.  Maybe "conditionally mutable - it allows for adding a second IPFamily..."